src/core/views/components/spinner.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
-import Spin from 'spin.js'
+import { Spinner as Spin } from 'spin.js'
```

Comment:
I noticed that the spinner was not spinning for me. Is this also happening for you guys?

Reply:
Yes, I see the same issue. I think that should be fixed.

Reply:
On it